code/script.js

```diff
@@ -34,18 +43,282 @@ const showMessage = (message, sender) => {
+//   } else if (gender === "gentleman" && selectedPriceRange === "10 to 50€" && selectedColour === "Yellow") 
+//   {showMessage ("How about a yellow baseball cap with a cool design or logo? This could be a practical and fashionable accessory.", 'bot')
+//   } else if (gender === "gentleman" && selectedPriceRange === "10 to 50€" && selectedColour === "Green") 
+//   {showMessage ("How about a green-themed notebook or journal? Those are great for jotting down thoughts, notes, or sketches.", 'bot')
+//   } else if (gender === "gentleman" && selectedPriceRange === "50 to 100€" && selectedColour === "Yellow") 
+//   {showMessage ("How about a yellow hoodie? It could be a comfortable and trendy addition to his casual wardrobe.", 'bot')
+//   } else if (gender === "gentleman" && selectedPriceRange === "50 to 100€" && selectedColour === "Green") 
+//   {showMessage ("How about a green backpack? Especially a durable and stylish one - it can be practical for everyday use or travel.", 'bot')
+//   }
+  
+// }
+
+
+//The bots suggestions based on users choices
+const suggestion = () => {
+  let suggestionMessage = '';
+
+  if (gender === 'lady') {
+    if (selectedPriceRange === '10 to 50€') {
+      if (selectedColour === 'Yellow') {
+        suggestionMessage = "How about a vibrant yellow mug? It can brighten her mornings with her favorite hot beverage.";
+      } else if (selectedColour === 'Green') {
+        suggestionMessage = "How about a potted green succulent? It is a low-maintenance and aesthetically pleasing gift.";
+      }
+    } else if (selectedPriceRange === '50 to 100€') {
+      if (selectedColour === 'Yellow') {
+        suggestionMessage = "How about a yellow toaster? It can add a pop of color to her kitchen.";
+      } else if (selectedColour === 'Green') {
+        suggestionMessage = "How about a soft green throw blanket? It could add comfort and style to her home.";
+      }
+    }
+  } else if (gender === 'gentleman') {
+    if (selectedPriceRange === '10 to 50€') {
+      if (selectedColour === 'Yellow') {
+        suggestionMessage = "How about a yellow baseball cap with a cool design or logo? This could be a practical and fashionable accessory.";
+      } else if (selectedColour === 'Green') {
+        suggestionMessage = "How about a green-themed notebook or journal? Those are great for jotting down thoughts, notes, or sketches.";
+      }
+    } else if (selectedPriceRange === '50 to 100€') {
+      if (selectedColour === 'Yellow') {
+        suggestionMessage = "How about a yellow hoodie? It could be a comfortable and trendy addition to his casual wardrobe.";
+      } else if (selectedColour === 'Green') {
+        suggestionMessage = "How about a green backpack? Especially a durable and stylish one - it can be practical for everyday use or travel.";
+      }
+    }
+  }
+
+  if (suggestionMessage) {
+    showMessage(suggestionMessage, 'bot');
```

Comment:
It's great that you have stored the message from the bot in an new separate variable!